docs/index.md

```diff
@@ -176,16 +245,17 @@ Devfile allows to specify commands set to be available for execution in workspac
    - name: build
      actions:
        - type: exec
-         tool: mysql
+         component: mysql
          command: mvn clean
          workdir: /projects/spring-petclinic
 ```
 
 ### Live working examples
 
-  - [NodeJS simple "Hello World" example](https://che.openshift.io/f?url=https://raw.githubusercontent.com/redhat-developer/devfile/master/samples/web-nodejs-sample/devfile.yml)
-  - [Java Spring-Petclinic example](https://che.openshift.io/f?url=https://raw.githubusercontent.com/redhat-developer/devfile/master/samples/web-java-spring-petclinic/devfile.yml)
-  - [Theia frontend plugin example](https://che.openshift.io/f?url=https://raw.githubusercontent.com/redhat-developer/devfile/master/samples/theia-hello-world-frontend-plugin/devfile.yml)
+  - [NodeJS simple "Hello World" example](https://che.openshift.io/f?url=https://raw.githubusercontent.com/redhat-developer/devfile/master/samples/web-nodejs-sample/devfile.yaml)
+  - [NodeJS Application with Mongo DB example](https://che.openshift.io/f?url=https://raw.githubusercontent.com/redhat-developer/devfile/master/samples/web-nodejs-with-db-sample/devfile.yaml)
+  - [Java Spring-Petclinic example](https://che.openshift.io/f?url=https://raw.githubusercontent.com/redhat-developer/devfile/master/samples/web-java-spring-petclinic/devfile.yaml)
+  - [Theia frontend plugin example](https://che.openshift.io/f?url=https://raw.githubusercontent.com/redhat-developer/devfile/master/samples/theia-hello-world-frontend-plugin/devfile.yaml)
 
 ### Planned features
-There is still a lot of plans to extend Devfile possibilities, such as support multiple kubernetes/openshift tools etc
+There is still a lot of plans to extend Devfile possibilities, such as support multiple dockerimage components etc
```

Comment:
> support multiple dockerimage components 

It's already implemented. Looks like no more plans to extend Devfile possibilities =)
Maybe it is better just to remove this section instead of keeping it updated?

Reply:
That should be done in devfile README. This doc is only mirror for it.